modules/RTC/RTC.js

```diff
@@ -864,8 +861,9 @@ export default class RTC extends Listenable {
             this._channel.close();
             this._channel = null;
 
-            this.removeListener(RTCEvents.LASTN_ENDPOINT_CHANGED,
-                this._lastNChangeListener);
+            this.removeListener(RTCEvents.LASTN_ENDPOINT_CHANGED, this._lastNChangeListener);
```

Comment:
why is the cleanup in both `destroy` method and here?

Reply:
Looks like the call in `destroy()` is redundant since `destroy()` is called after `closeBridgeChannel()` when client leaves the room. I will delete the call there. Thanks for pointing it out.